adapters/balance.js

```diff
@@ -38,6 +38,14 @@ module.exports = function (mu, transports) {
 
 
 
+  function setMu (muInstance) {
+    transports.forEach(function (transport) {
+      transport.setMu(muInstance)
+    })
+  }
```

Comment:
I think it is necessary for the transport to hold a reference to the mu scope, however this is an internal detail which may change